dockers/docker-snmp/snmp_yml_to_configdb.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/lib/env python
+
+import yaml
+from swsssdk import ConfigDBConnector, SonicV2Connector, SonicDBConfig
```

Comment:
It looks like you're only using `ConfigDBConnector`. I think you can remove `SonicV2Connector` and `SonicDBConfig`

Reply:
fixed.